### ISSUE TITLE
"Visible to"-option for final versions

### ISSUE THREAD

**Issue by jpennekamp** [kohler/hotcrp#231]
Please correct me if I'm mistaken. Currently, there seems to be no way to hide final versions from (external) reviewers, which is arguably a special use case.

In the "submission form" settings, the visibility of each newly added field can be adjusted using the "visible to" option. However, for the "final versions" enabled under "decision", no such option exists. While a workaround could be to collect the final versions through a newly added field instead, having the option to specify the visibility of final versions would be great.

The following use case comes to mind:
- The reviewing phase is double-blind, i.e., submitted papers have no author information
- The final submission is only single-blind (includes author names). Without adjusting the field's visibility, reviewers have implicitly access to the author names through the collected final version.

**TL;DR:** Having the same "visible to" option for final versions as for newly added submission fields would be desirable.
